resource_activity/models/resource_activity.py

```diff
@@ -678,11 +677,18 @@ def prepare_lines(self, activity):
 
     def _create_sale_order(self, activity, partner_id):
         SaleOrder = self.env['sale.order']
+        sale_note = SaleOrder._default_note()

```

Comment:
what _default_note() is used for ?

Reply:
[it fetches the `sale_note` defined on res.company level (standard)](https://github.com/odoo/odoo/blob/9.0/addons/sale/sale.py#L93)